docs/BreakingChanges/095 ConcurrentQueue.TryPeek can return an erroneous null via its out parameter.md

```diff
@@ -0,0 +1,25 @@
+## 95: ConcurrentQueue<T>.TryPeek can return an erroneous null via its out parameter
```

Comment:
{
  "Id": "95",
  "Title": "ConcurrentQueue<T>.TryPeek can return an erroneous null via its out parameter",
  "Details": "In some multi-threaded scenarios, `ConcurentQueue<T>.TryPeek` can return true, but populate the out parameter with a null value (instead of the correct, peeked value).",
  "Suggestion": "This issue is fixed in the .NET Framework 4.5.1. Upgrading to that Framework will solve the issue.",
  "Link": "http://connect.microsoft.com/VisualStudio/feedback/details/762273/bug-in-concurrentqueue-in-net-4-5-trypeek-returns-true-but-no-real-object-returned",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Collections.Concurrent.ConcurrentQueue`1.TryPeek(`0@)"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": "4.5.1",
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": true,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 1
}

**Parsed APIs:**
- [x] M:System.Collections.Concurrent.ConcurrentQueue`1.TryPeek(`0@)